client/src/test/runTests.ts

```diff
@@ -11,11 +11,15 @@ async function main() {
         // Passed to --extensionTestsPath
         const extensionTestsPath = path.resolve(__dirname, '../../node_modules/vscode-jest-test-runner');
 
+        const resourcesDirPath = path.resolve(extensionDevelopmentPath, 'client/resources');
+        const kbDirPath = path.resolve(resourcesDirPath, 'kb');
+        const documentFormatFilePath = path.resolve(kbDirPath, '_technical/documentFormat.dcl');
+
         // Download VS Code, unzip it and run the integration test
         await runTests({
             extensionDevelopmentPath,
             extensionTestsPath,
-            launchArgs: ['--disable-extensions'],
+            launchArgs: [`${kbDirPath}/`, documentFormatFilePath, '--disable-extensions'],
```

Comment:
* Not exactly what I had in mind xd
I asked if there could be two arguments instead of three, letting the test files themselves deduce `documentFormatFilePath` from `kbDirPath`. But maybe this way we’re more free (like, we can decorrelate the two paths in case `documentFormatFilePath` stops being under `kbDirPath`). :woman_shrugging: 

* Not sure why we’d need to append a slash to the directory path. That’s typically the type of things I try to avoid in Bash to make it simpler to concatenate stuff. I only append slashes for some logs, not in the actual variable values. But maybe here there’s some kind of obscure reason?

Reply:
The first arguments make VSCode open a directory or a file, depending on the end of the path. If the path ends with a slash, it is a directory (opening therefor some kind of workspace). If not, it will open or create a file, depending on whether the file exists or not.
I want to start my tests with a directory opened and a file open too.

Reply:
Oh, crap. But that’s weird.

> If not, it will open or create a file, depending on whether the file exists or not.

Since it’s impossible to create a `foo` file when there’s already a `foo/` directory, a simple check would be able to make that “slash vs. no slash” distinction useless. Dunno what’s going on with that stuff.